external/fv3fit/fv3fit/emulation/keras.py

```diff
@@ -1,10 +1,11 @@
 import dataclasses
```

Comment:
Maybe move to a new module named `jacobian.py` instead. `keras.py` is a pretty random assortment of things.